### ISSUE TITLE
Add ARIA attributes to collapsed navbar (similar to dropdown)

### ISSUE THREAD

**Issue by jmuheim** [twbs/bootstrap#16099]
While the [dropdown](http://getbootstrap.com/components/#dropdowns) has nice accessibility support using ARIA attributes (e.g. `aria-expanded`), the [navbar](http://getbootstrap.com/components/#navbar)'s "Toggle navigation" button (only displayed on small screen resolutions) doesn't have any ARIA attributes.

This is necessary for people on mobile devices who are using screenreaders (e.g. VoiceOver on iOS).

**Comment by patrickhlauke** [comment]
Funnily enough, I was looking at that exact thing this morning (as part of #16096). Leave it with me :)
